packs/age-of-ashes-bestiary/barushak-il-varashma.json

```diff
@@ -1733,7 +1733,7 @@
                 "damage": {},
                 "defense": null,
                 "description": {
-                    "value": "<p>You augment the abilities of a summoned creature. The target gains a +1 status bonus to all checks (this also applies to the creature's DCs, including its AC) for the duration of its summoning, up to 1 minute.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Augment Summoning]</p>"
+                    "value": "<p>You augment the abilities of a summoned creature. The target gains a +1 status bonus to all checks (this also applies to the creature's DCs, including its AC) for the duration of its summoning, up to 1 minute.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Fortify Summoning]{Spell Effect: Augment Summoning}</p>"
```

Comment:
Got some cleanup there

Reply:
Leaving the old name intact was intentional - should I strip it?